src/components/Header/WalletButton.tsx

```diff
@@ -31,13 +65,27 @@ const WalletButton = () => {
       sx={{
         gridArea: "wallet",
         marginLeft: "auto",
-        marginRight: "0.5rem",
+        marginRight: 0,
         display: "flex",
         alignItems: "center",
       }}
     >
-      {accountId && <Box sx={{ fontSize: "0.85rem", mr: "1rem" }}>NEAR: {balance}</Box>}
-      <Box>
+      {accountId ? (
+        <Box
+          sx={{
+            marginTop: 0.6,
+            fontSize: "0.8rem",
+            display: "flex",
+            flexFlow: isMobile ? "column" : "row",
+          }}
+        >
+          <div style={{ fontWeight: "bold" }}>{accountTrim(accountId)}</div>
+          <div style={{ display: "flex", justifyContent: "flex-end", marginLeft: 4 }}>
+            {balance.substring(0, balance.length - 2)}
```

Comment:
this does not take into account balances without decimals, like 1 NEAR
you can make a new TOKEN_FORMAT with only 2 decimals

Reply:
great point!

Reply:
gonna use: `Number.parseFloat(balance).toFixed(2)`